apps/src/code-studio/pd/form_components/FormComponent.jsx

```diff
@@ -80,9 +97,10 @@ export default class FormComponent extends React.Component {
         onChange={this.handleChange}
         value={this.props.data[name] || ''}
         required={required}
+        {...props}
       >
-        {placeholder && <option key="placeholder">{placeholder}</option>}
-        {options}
+        {placeholder && !this.props.data[name] && <option key="placeholder">{placeholder}</option>}
```

Comment:
I think we still want the placeholder if they've already answered, so they can clear their answer. I suppose we _could_ do this if `required === false`, but only then.